<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetBottom</class>
 <widget class="QWidget" name="WidgetBottom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>170</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>170</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelKingdom">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>6</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHands">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>56</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelHandsNumber">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>101</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="LabelClickable" name="labelRole">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>58</y>
     <width>25</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelected">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>31</y>
     <width>134</width>
     <height>134</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelSelected</zorder>
  <zorder>labelKingdom</zorder>
  <zorder>labelHands</zorder>
  <zorder>labelHandsNumber</zorder>
  <zorder>labelRole</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelClickable</class>
   <extends>QLabel</extends>
   <header>ui/others/labelclickable.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
